sources/reference/admiral.md

```diff
@@ -90,7 +107,23 @@ Confirm that the required values have been entered correctly. If you have decide
 
 Admiral will then install dependencies, pull Docker images, and install the database. This may take several minutes. When it is complete, the location of the Admiral UI and the login token are printed. Navigate to the UI location in a web browser and enter the given access key to access the installer UI. The rest of the services will be started from there.
 
-### Info
+### upgrade
+This command will upgrade the Shippable installation. To list the available
+versions, run `git tag` command inside `admiral` directory. To install
+Shippable version corresponding to any tag, run `git checkout <tag>` and then
+run `./admiral.sh upgrade` command.
+All the Shippable services will be upgraded to the new tag once the command is
```

Comment:
This is a little awkward to read.  How about "all Shippable services will be upgraded to the new tag"?  It's already the section about the upgrade command, so we can assume that anything here is for that command.